bin/cmd.cjs

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
+
+require('version-guard')('../lib/cli.js', 12, 22)
```

Comment:
Does that mean, that for Node.js version `< 12.22`, we fail silently?
What happens, if we run it, with for example Node.js v13, a Node.js version `> 12.22` but that we don't support as per `engines.node` property in `package.json`, ref: https://github.com/standard/standard/blob/8d832307d9f4e5875c7c32487e596d29db2cf10f/package.json#L37

Reply:
That would go beyond fixing the/hardening the current guard so i would say its out of scope, especially considering the mixed stances on whether version guards at all are a good idea

Reply:
Sounds good to me, considering the behavior is the same as before. :+1: